src/plugin/connector/loadbalancer/listener_connector.py

```diff
@@ -26,3 +26,17 @@ def get_listener(self, secret_data: dict, region: REGION) -> dict:
             raise Exception(f"Failed to get LB Listeners. {response.json()}")
 
         return response.json().get("listeners", [])
+
+    def get_listener_detail(self, listener_id: str, secret_data: dict, region: REGION) -> dict:
+        token = self.get_token(secret_data)
+
+        url = f"https://{region.name.lower()}-api-network-infrastructure.nhncloudservice.com/v2.0/lbaas/listeners/{listener_id}"
+        headers = {"X-Auth-Token": token}
+
+        response = requests.get(url, headers=headers)
+
+        if response.status_code != 200:
+            _LOGGER.error(f"Failed to get LB Listeners Detail. {response.json()}")
+            raise Exception(f"Failed to get LB Listeners Detail. {response.json()}")
+
+        return response.json().get("listener", [])
```

Comment:
According to line 30, this function has to return dictionary. But when failed to get "listener" key, it returns array.